api/nodes/views.py

```diff
@@ -1551,8 +1551,8 @@ def get_queryset(self):
             self.get_provider_item(addon, node=node)
             for addon
             in node.get_addons()
-            if addon.config.has_hgrid_files
-            and addon.configured
+
+            if getattr(addon.config, 'has_hgrid_files', True) and addon.configured
```

Comment:
nit/noop: could alternatively just define `has_hgrid_files` to return a value in the `EphemeralNodeConfig`?